src/Learning/KWUtils/KWKhiopsVersion.h

```diff
@@ -10,7 +10,7 @@
 // dans le TaskManager de Windows (par exemple)
 
 // Version de Khiops
-#define KHIOPS_VERSION KHIOPS_STR(10.2.2)
+#define KHIOPS_VERSION KHIOPS_STR(10.3.0-a.4)
```

Comment:
Why are we incrementing the minor? I think these changes are patches.

Reply:
Yes, this is just a temporary tag for tests only. I think that the "final" tag would depend on where this change would be put on the release roadmap.

Reply:
Ok

Reply:
This should land into 10.2.3 IMHO (if LearningTest reports are OK for all supported platforms).

Reply:
Tagged with 10.2.3-b.1